src/frontend/components/Felleskomponenter/Datovelger/Datovelger.test.tsx

```diff
@@ -71,11 +76,14 @@ describe('Datovelger', () => {
             </TestProvidere>
         );
 
-        const tilOgMedÅpneknapp = getAllByRole('button')[1];
+        const [, tilOgMedÅpneknapp] = getAllByRole('button');
         act(() => tilOgMedÅpneknapp.click());
-        const forrigeDag = container.querySelector('[aria-label="torsdag 9"]');
 
-        expect(forrigeDag?.getAttribute('disabled')).not.toBeNull();
+        const [tilOgMedKalender] = getAllByRole('dialog');
```

Comment:
Tidligere gikk vi ut fra at `container.querySelector('[aria-label="torsdag 9"]');` kun ville finne datoen i kalender for til-og-med, men nå som Datepicker rendrer kalenderen til DOM uansett om den vises for brukeren vil det første treffet være fra fra-og-med-kalenderen, der datoen er tillatt å velge.

Skriver om til at vi henter ut til-og-med-kalenderen ved å se etter dialog-elementer som er tilgjengelige for brukeren (i dette caset er fra-og-med-kalenderen aria-hidden og treffer derfor ikke på `getAllByRole('dialog')`)